### PR TITLE
ft #163557475

### PR DESCRIPTION
**Pull request by chokonaira**
#### What does this PR do?
Fixed politicoApp UI responsiveness

#### Description of Task to be completed?
Politico app can now be accessed by mobile users

#### How should this be manually tested?
None
#### Any background context you want to provide?
Media queries

#### What are the relevant pivotal tracker stories?
#163557475
#### Questions:

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "airbnb-base"
+}
```
Comment (tutugodfrey) [inline .eslintrc.json]: I don't think you need this file on this branch.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  "devDependencies": {
+    "@babel/core": "^7.2.2",
+    "@babel/node": "^7.2.2",
+    "eslint": "^5.12.1",
+    "jasmine": "^3.3.1",
+    "mocha": "^5.2.0",
+    "nodemon": "^1.18.9"
+  }
+}
```
Comment (tutugodfrey) [inline package.json]: I don't think you need these file on this branch.

File: server.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+const port = process.env.PORT || 3000;
+const app = express();
+
+app.get('/', (req, res, next) => {
+  res.send('Hello World');
+})
+app.listen(PORT, () => {
+  console.log(`server running on port ${port}`);
+});
+
```
Comment (tutugodfrey) [inline server.js]: I don't think you need these file on this branch.

File: .babelrc [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    "react",
+    "stage-0"
+  ],
+  "plugins": [
+    "transform-class-properties",
+    "transform-decorators",
+    "transform-react-constant-elements",
+    "transform-react-inline-elements"
+  ]
+}
```
Comment (tutugodfrey) [inline .babelrc]: @chokonaira great job coming up with the modifications. I don't think you need these file on this branch.

File: .eslintrc.json (hunk 1 above)
Comment (chokonaira) [reply .eslintrc.json]: Ok Godfrey, i am presently working on fixing it. Thank you

**Pull request closed without merging**